test/jasmine/assets/scatter_cluster_mapbox_test.js

```diff
@@ -0,0 +1,176 @@
+var Plots = require('@src/plots/plots');
+var Lib = require('@src/lib');
+var Axes = require('@src/plots/cartesian/axes');
+
+var ScatterClusterMapbox = require('@src/traces/scatterclustermapbox');
```

Comment:
A trace titled `scatterclustermapbox` no longer exists.
Please integrate these tests with `scattermapbox` tests.
Thank you!

Reply:
@elben10  could explain why the interactive tests are removed in https://github.com/plotly/plotly.js/pull/5147/commits/1f42ee217b321134740505010e374f707a27c1d1?

Reply:
The tests should be integrated in [`scattermapbox_test.js`](https://github.com/plotly/plotly.js/blob/master/test/jasmine/tests/scattermapbox_test.js) now where scattermapboxcluster doesn't exist as a separate trace anymore. Or am I missing something? (I am aware, that these test is not integrated yet)

Reply:
> The tests should be integrated in [`scattermapbox_test.js`](https://github.com/plotly/plotly.js/blob/master/test/jasmine/tests/scattermapbox_test.js) now where scattermapboxcluster doesn't exist as a separate trace anymore. Or am I missing something? (I am aware, that these test is not integrated yet)

Thanks very much for the note and hard work on this feature. 
This PR seems almost complete with jasmine tests.